Source: written by Pierre Lejeune, for Exherbo
Upstream: submitted, https://bugzilla.samba.org/show_bug.cgi?id=9018
Reason: Fix python module for correct detection by samba:4 build system

--- a/lib/ntdb/pyntdb.c	2012-06-30 22:43:59.640003079 +0200
+++ b/lib/ntdb/pyntdb.c	2012-06-30 22:44:34.580005168 +0200
@@ -550,8 +550,8 @@
 	{ NULL }
 };
 
-void inittdb(void);
-void inittdb(void)
+void initntdb(void);
+void initntdb(void)
 {
 	PyObject *m;
 
